packages/mdc-notched-outline/index.js

```diff
@@ -39,15 +40,30 @@ class MDCNotchedOutline extends MDCComponent {
   static attachTo(root) {
     return new MDCNotchedOutline(root);
   }
+  /** @param {...?} args */
+  constructor(...args) {
+    super(...args);
+    /** @private {Element} */
+    this.notchElement_;
+  }
+
+  initialSyncWithDOM() {
+    const label = this.root_.querySelector('.' + MDCFloatingLabelFoundation.cssClasses.ROOT);
+    this.notchElement_ = this.root_.querySelector(strings.NOTCH_ELEMENT_SELECTOR);
+
+    if (label) {
+      label.style.transitionDuration = '0s';
+      label.classList.add(MDCFloatingLabelFoundation.cssClasses.UPGRADED);
```

Comment:
This seems odd... modifying the class list of a floating label from within notched outline code. If this logic needs to be within notched-outline, I would suggest we apply the upgraded class to notched-outline and adjust CSS selectors accordingly.